data/mods/NW_Pack/nw_books.json

```diff
@@ -17,5 +17,24 @@
     "skill": "fabrication",
     "price": 30000,
     "required_level": 8
+  },
+  {
+    "type": "BOOK",
+    "id": "book_mythological",
+    "name": "The Weapons of Asgard and Beyond",
+    "name_plural": "The Weapons of Asgard and Beyond",
+    "max_level": 10,
+    "description": "This book is about creating replicas of the weapons used by the Norse Gods, such as the well-known Mjölnir, and Gungnir.  It is very well illustrated with lots of pictures and norse text, no doubt an enchantment.",
```

Comment:
This seems awkwardly worded. Maybe something like this?

This ornate book appears to be written in some sort of Norse text. Based on the detailed illustrations, it must contain information about crafting replicas of weapons used by the Norse Gods.

Reply:
I agree, the book was a last minute thought, only when I realized there was no way for the weapons to be a thing without it.